src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -97,3 +102,43 @@ case class BreakdownResult(
     delta: Double)
 
 case class Failure(className: String, message: String)
+
+// KEEP ARGUMENTS SORTED BY NAME.
+// It simplifies lookup when checking if a parameter is here already.
+case class MLParams(
+    // *** Common to all algorithms ***
+    randomSeed: Option[Int] = None,
```

Comment:
It would be nice to always store the seed in results.  How about making the default value be Some(42)?